src/modules/message-parsers.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Incoming message parser - parse message based on headers
+ * @param  {object} _msg An amqp.node incoming message
+ * @return {any}      a string, object, number to send. Something stringifiable
+ */
+module.exports.in = (_msg) => {
+  //if sender put a json header, we parse it to avoid the pain for the consumer
+  if (_msg.properties.contentType === 'application/json') {
+    try {
+      return JSON.parse(_msg.content.toString());
+    } catch(e) {}
```

Comment:
Is just swallowing the error ok here, any side effects?  Maybe logging it just to be aware ?